products/sle/main.pm

```diff
@@ -1007,6 +1007,17 @@ else {
             loadtest "console/yast2_nfs_client";
         }
     }
+    elsif (get_var("NFS4SERVER") or get_var("NFS4CLIENT")) {
```

Comment:
```suggestion
    elsif (get_var("NFS4SERVER") || get_var("NFS4CLIENT")) {
```
I had some weird behavior of `or` in perl . Don't remember exact details but I just know that you need to always prefer `||` unless you really know what you are doing